catalog/cataloger_commit.go

```diff
@@ -108,7 +109,7 @@ func commitDeleteUncommittedTombstones(tx sqlx.Execer, branchID int64, commitID
 }
 
 func commitTombstones(tx sqlx.Execer, branchID int64, commitID CommitID) (int64, error) {
```

Comment:
I think commitTombstones is redundent. Just remove the condition of "AND NOT is_deleted" from CommitEntries